packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/12-connection-unstable.mdx

```diff
@@ -1,4 +1,75 @@
 ---
+id: connection-unstable
 title: Connection Unstable
 description: Connection Unstable
----
\ No newline at end of file
+---
+
+Besides showing the quality of user's network connection, another useful video call feature is to indicate unreliable participant's connection quality. The [default `ParticipantViewUI`](../../ui-components/core/participant-view) does this by rendering [`Notification`](todo docs link) component when the connection quality reaches poor level.
```

Comment:
`Notification` component won't be in our docs, context: https://github.com/GetStream/stream-video-js/pull/558